src/main/java/ui/DirectoryController.java

```diff
@@ -0,0 +1,40 @@
+package ui;
+
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.control.TableCell;
+import javafx.scene.control.TableColumn;
+import javafx.scene.control.TextField;
+
+import java.io.IOException;
+
+/**
+ * Created by Ari on 2/6/17.
+ */
```

Comment:
Might want to be removed, but this isn't something so critical as to delay this pull request.